### DISCUSSION TITLE
ppocrlabel程序进行python打包后，问题逐一解决后，最后出现的问题很头痛

### DISCUSSION THREAD

**Discussion by LXX230** [PaddlePaddle/PaddleOCR#12393] [Q&A]
Traceback (most recent call last):
  File "PPOCRLabel\PPOCRLabel.py", line 2837, in <module>
AssertionError: operate the cmd have some problems ,please check  whether there is a in the lib directory resources.py 
这段是运行PPOCRLabel.exe后出现的报错弹窗。

pyrcc5: File does not exist 'resources.qrc' 这一段是cmd闪一下后我瞬间截的图

搞了一天不知道问题出在哪，请各位大佬多多指点

**Comment by Hommoner** [comment]
試試看把那段程式碼拿掉然後放編譯好的resources.qrc 檔案在你的exe目錄下
```
if __name__ == '__main__':
    '''
    resource_file = './libs/resources.py'
    if not os.path.exists(resource_file):
        output = os.system('pyrcc5 -o libs/resources.py resources.qrc')
        assert output == 0, "operate the cmd have some problems ,please check  whether there is a in the lib " \
                            "directory resources.py "
    '''                        
    sys.exit(main())
```

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/50170626/e97d6729-461f-4862-92ab-797fab35a83f)

**Comment by fenghuoxiguozu** [comment]
同样的问题，有解决方式吗

**Reply by liuhongen1234567** [reply]
您好，似乎执行下面的命令可以解决，具体可以参考这个issue https://github.com/HumanSignal/labelImg/issues/568
```python
pyrcc5 -o line/resources.py resources.qrc
```
